src/app/home/home.component.html

```diff
@@ -28,11 +35,13 @@
           <button mat-flat-button color="primary" (click)="login()" class="custom mb-1">
             Log in
           </button>
+          -->
           <!-- <span> or </span>
                     <button mat-stroked-button color="primary" (click)="register()" class="custom mb-1">
                         Register an account
                     </button> -->
-          <a mat-stroked-button color="primary" class="custom no-underline ml-1" target="_blank" [href]="documentationUrl">Learn more</a>
```

Comment:
Log in and learn more buttons on centre of the front page have been commented out.

Reply:
Both these issues fixed with 6bff91868e50914faa09a5f037f05334b6db35be